### ISSUE TITLE
Java: filterGraphQLErrors is not in SimpleGraphQLServlet

### ISSUE THREAD

**Issue by gadogttas** [howtographql/howtographql#100]
The way it's written and presented in the 'Error Handling' section, (https://www.howtographql.com/graphql-java/7-error-handling/) we're supposed to be able to override filterGraphQLErrors in GraphQLEndpoint. However, that method is actually found in DefaultGraphQLErrorHandler.java instead. One way to fix this would be to introduce a new ErrorHandler for our project and override it there.

**Comment by kaqqao** [comment]
This is true in graphql-java-servlet 4.1.0, which got released a couple of days ago. In 4.0.0, which the tutorial is using, the given code is correct.

While I will try keeping the tutorial up-to-date, this can only be on best-effort basis.

**Comment by gadogttas** [comment]
Ok, that makes sense. Maybe the recommendation to update to the newest
version of things should be removed from the beginning of the tutorial
then? I'm loving these resources, don't get me wrong. I'm just providing
feedback. :)

**Comment by kaqqao** [comment]
No worries, I appreciate the help! I've already updated the the examples to 4.1.0 locally. Will push once I test everything.

As for the advice to update... I was thinking of removing it too. It's a good idea to update, but it certainly might put you in exactly this situation.

**Comment by marktani** [comment]
I think it makes sense to do a version upgrade every few weeks/months and to stick to one version in between 🙂

**Comment by nikolasburk** [comment]
Is this issue resolved @kaqqao and @gadogttas?

**Comment by kaqqao** [comment]
It's not. I didn't have the time to maintain the examples. And now I think I must rewrite the tutorial as the landscape changed too much in the meantime. I no longer recommend graphql-java-tools at all as graphql-java itself now has all the features needed.

**Comment by kaqqao** [comment]
I'm planning to start the work on it soon. Perhaps this weekend.
